packages/web-api-scan-runner/src/report-generator/axe-result-to-html-converter.ts

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { AxeReportParameters, ReporterFactory } from 'accessibility-insights-report';
+import { AxeResults } from 'axe-core';
+import { inject, injectable } from 'inversify';
+import { ReportFormat } from 'storage-documents';
+import { iocTypeNames } from '../ioc-types';
+import { AxeResultConverter, ReportGenerationParams } from './axe-result-converter';
+
+@injectable()
+export class AxeResultToHtmlConverter extends AxeResultConverter {
+    public readonly reportType: ReportFormat = 'html';
```

Comment:
This is not used here. Can we have public type of report formats declared outside so it can be referenced in a solution?

Reply:
resolved via offline discussion